### PR TITLE
checkstyles and graph additions

### PR DESCRIPTION
**Pull request by YaffaLevy**
Screen Shot 2024-02-28 at 4 35 46 PM

### CODE REVIEW

File: src/main/java/levy/projectile/ProjectileGraph.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        g.fillRect(200, -200, 50, 50);
+        g.setColor(Color.ORANGE);
+        g.drawOval(200, -200, 50, 50);
+
+        g.fillOval(
+                (int) projectile.getX(),
+                (int) -projectile.getY(),
+                10,
+                10
+        );
```
Comment (andrewoid) [inline src/main/java/levy/projectile/ProjectileGraph.java]: This code doesn't match your screenshot.

**Comment by YaffaLevy** [comment]
[image: Screen Shot 2024-02-28 at 4.04.49 PM.png]

i tried to push but this happened

File: src/main/java/levy/projectile/ProjectileGraph.java [hunk 2]
```diff
@@ -5,31 +5,32 @@
...
-
-        g.fillOval(
-                (int) projectile.getX(),
-                (int) -projectile.getY(),
-                10,
-                10
-        );
-    }
+        g.setColor(Color.BLACK);
+        g.fillOval((int) projectile.getInterceptX() / 2, (int) -projectile.getPeakY(), 10, 10);
```
Comment (andrewoid) [inline src/main/java/levy/projectile/ProjectileGraph.java]: oval needs to be centered on the curve.

**Comment by andrewoid** [comment]
Replace the screenshot in the description of the PR

**Comment by YaffaLevy** [comment]
Screen Shot 2024-02-28 at 4 29 46 PM

**Comment by andrewoid** [comment]
> Replace the screenshot in the description of the PR

Read this comment again and follow what I said.

File: src/main/java/levy/projectile/ProjectileFrame.java [hunk 3]
```diff
@@ -1,31 +1,39 @@
 package levy.projectile;
 
 import javax.swing.*;
+import javax.swing.event.DocumentEvent;
 import java.awt.*;
 
 
 public class ProjectileFrame extends JFrame {
 
+    static int fpsMin = 0;
```
Comment (andrewoid) [inline src/main/java/levy/projectile/ProjectileFrame.java]: `final`

File: src/main/java/levy/projectile/ProjectileGraph.java [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+            g.setColor(Color.BLACK);
+            nowX = (int) projectile.getX();
+            nowY = (int) projectile.getY();
+            projectile.setSeconds(t);
+            g.drawLine(nowX, -nowY, (int) projectile.getX(), (int) -projectile.getY());
+
+        }
+
+    }
+    public void setProjectile(Projectile projectile) {
```
[the remark this answers is not in the document]
Comment (andrewoid) [reply src/main/java/levy/projectile/ProjectileGraph.java:32]: You need a newline in between lines 30 and 31

**Review by andrewoid — APPROVED** [review]

**Pull request merged**